README.md

```diff
@@ -118,19 +131,44 @@ Environment variable    | Description | Required | Example value
 `VAULT_RATE_LIMIT`      | Only applies to a single invocation of the extension. See [Vault Commands (CLI)][vault-env-vars] documentation for details. Ignored by proxy server | No | `10`
 `VAULT_NAMESPACE`       | The namespace to use for pre-configured secrets. Ignored by proxy server | No | `education`
 
+### AWS STS client configuration
+
+In addition to Vault configuration, you can configure certain aspects of the STS
+client the extension uses through the usual AWS environment variables. For example,
+if your Vault instance's IAM auth is configured to use regional STS endpoints:
+
+```bash
+vault write auth/aws/config/client \
+  sts_endpoint="https://sts.eu-west-1.amazonaws.com" \
+  sts_region="eu-west-1"
+```
+
+Then you may need to configure the extension's STS client to also use the regional
+STS endpoint by setting `AWS_STS_REGIONAL_ENDPOINTS=regional`, because both the AWS Golang
+SDK and Vault IAM auth method default to using the global endpoint in many regions.
+See documentation on [`sts_regional_endpoints`][lambda-sts-regional-endpoints] for more information.
+
 ## Limitations
 
-For this early release, the extension does not support automatic secret renewal.
-This means once a secret is written to disk, it will not be refreshed once it
-expires. This may cause problems if you use [provisioned concurrency][lambda-provisioned-concurrency]
-or if your Lambda is invoked often enough that execution contexts live beyond
-the lifetime of the secret.
+Secrets written to disk or returned from the proxy server will not be automatically
+refreshed when they expire. This is particularly important if you configure the
+extension to write secrets to disk, because the extension will only write to disk
+once per execution environment, rather than once per function invocation. If you
+use [provisioned concurrency][lambda-provisioned-concurrency] or if your Lambda
+is invoked often enough that execution contexts live beyond the lifetime of the
+secret, then secrets on disk are likely to become invalid. In line with
+[Lambda best practices][lambda-best-practices], we recommend avoiding writing
+secrets to disk where possible, and exclusively consuming secrets via the proxy
+server.
```

Comment:
I think it's implied that the proxy server method will refresh secrets here? This paragraph doesn't directly say that though, and mentions secrets returned from proxy method, so it's confusing to me if those secrets are refreshed or not? 

Are they refreshed when requested again?

Reply:
Yeah I struggled to explain this clearly. I've given it another pass, LMK what you think.